tests/Resources/util.test.js

```diff
@@ -0,0 +1,1551 @@
+		});
+
+		it('with default breakLength just below break point', () => {
+			const obj = {
+				foo: '',
+				foobar: 1,
+				something: '1',
+				whatever: '',
+				whatever2: ''
+			};
+			util.inspect(obj).should.eql('{ foo: \'\', foobar: 1, something: \'1\', whatever: \'\', whatever2: \'\' }');
+		});
+	});
+
+	describe('#inherits()', () => {
+		it('is a function', () => {
+			util.inherits.should.be.a.Function;
+		});
+
+		it('hooks subclass to super constructor', (finished) => {
+			function BaseClass() {
+				this.listeners = {};
+			}
+
+			BaseClass.prototype.on = function (eventName, listener) {
+				const eventListeners = this.listeners[eventName] || [];
+				eventListeners.push(listener);
+				this.listeners[eventName] = eventListeners;
+			};
+
+			BaseClass.prototype.emit = function (eventName, data) {
+				const eventListeners = this.listeners[eventName] || [];
+				for (const listener of eventListeners) {
+					listener.call(this, data);
+				}
+			};
+
+			function MyStream() {
+				BaseClass.call(this);
+			}
+
+			util.inherits(MyStream, BaseClass);
+
+			MyStream.prototype.write = function (data) {
+				this.emit('data', data);
+			};
+
+			const stream = new MyStream();
+
+			should(stream instanceof BaseClass).eql(true);
+			should(MyStream.super_).eql(BaseClass);
+
+			stream.on('data', data => {
+				data.should.eql('It works!');
+				finished();
+			});
+			stream.write('It works!'); // Received data: "It works!"
+		});
+
+		it('throws TypeError if super constructor is null', () => {
+			function BaseClass() {
+			}
+
+			function MyStream() {
+				BaseClass.call(this);
+			}
+
+			should.throws(() => util.inherits(MyStream, null),
+				TypeError
+			);
+		});
+
+		it('throws TypeError if constructor is null', () => {
+			function BaseClass() {
+			}
+
+			should.throws(() => util.inherits(null, BaseClass),
+				TypeError
+			);
+		});
+
+		it('throws TypeError if super constructor has no prototype', () => {
+			const BaseClass = Object.create(null, {});
+
+			function MyStream() {
+				BaseClass.call(this);
+			}
+
+			should.throws(() => util.inherits(MyStream, BaseClass),
+				TypeError
+			);
+		});
+	});
+
+	describe('#isArray', () => {
+		it('should return true only if the given object is an Array', () => {
+			should.strictEqual(util.isArray([]), true);
+			should.strictEqual(util.isArray(Array()), true);
+			should.strictEqual(util.isArray(new Array()), true);
+			should.strictEqual(util.isArray(new Array(5)), true);
+			should.strictEqual(util.isArray(new Array('with', 'some', 'entries')), true);
+			should.strictEqual(util.isArray({}), false);
+			should.strictEqual(util.isArray({ push: function () {} }), false);
+			should.strictEqual(util.isArray(/regexp/), false);
+			should.strictEqual(util.isArray(new Error()), false);
+			should.strictEqual(util.isArray(Object.create(Array.prototype)), false);
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/u...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/util.test.js;
  Line: 848;
-->

- :warning: tests/Resources/util.test.js line 848 – 'util.isArray' was deprecated since v4.0.0. Use 'Array.isArray()' instead. (node/no-deprecated-api)